src/Umbraco.Web/PropertyEditors/ValueConverters/BlockListPropertyValueConverter.cs

```diff
@@ -51,66 +49,70 @@ public override object ConvertIntermediateToObject(IPublishedElement owner, IPub
 
             using (_proflog.DebugDuration<BlockListPropertyValueConverter>($"ConvertPropertyToBlockList ({propertyType.DataType.Id})"))
             {
-                var configuration = propertyType.DataType.ConfigurationAs<BlockListConfiguration>();
-                var blockConfigMap = configuration.Blocks.ToDictionary(x => x.ContentElementTypeKey);
-                var validSettingElementTypes = blockConfigMap.Values.Select(x => x.SettingsElementTypeKey).Where(x => x.HasValue).Distinct().ToList();
-
-                var contentPublishedElements = new Dictionary<Guid, IPublishedElement>();
-                var settingsPublishedElements = new Dictionary<Guid, IPublishedElement>();
-
-                var layout = new List<BlockListItem>();
```

Comment:
I've moved this below the short-circuiting code to prevent doing unnecessary work.